CHANGELOG.md

```diff
@@ -133,6 +137,7 @@ Added `compare-consensuses.py` script.
 ## 3.0.32 July 6, 2018
 
 Added `bin/newick-to-ascii.py` script.
+>>>>>>> master
```

Comment:
Is this line from a conflict that hasn't been resolved?

Reply:
Ah thanks, it is from a resolved conflict. Removed.